Aztec/Classes/Public/TextKit/AztecTextAttachment.swift

```diff
@@ -40,22 +35,80 @@ public class AztecTextAttachment: NSTextAttachment
         super.init(coder: aDecoder)
     }
 
+    // MARK: - Origin calculation
+
+    func xPosition(forContainerWidth containerWidth: CGFloat) -> Int {
+
```

Comment:
I've noticed we've got this extra spacing ... all over. Is this something we want / our standard, or it's just by chance?